internal/driver/cli.go

```diff
@@ -364,5 +364,7 @@ var usageMsgVars = "\n\n" +
 	"   PPROF_BINARY_PATH  Search path for local binary files\n" +
 	"                      default: $HOME/pprof/binaries\n" +
 	"                      searches $buildid/$name, $buildid/*, $path/$buildid,\n" +
-	"                      ${buildid:0:2}/${buildid:2}.debug, $name, $path\n" +
+	"                      ${buildid:0:2}/${buildid:2}.debug, $name, $path,\n" +
+	"                      ${name}.debug, $path/.debug/${name}.debug,\n" +
```

Comment:
$path should be $dir here I think. Because two lines above $path is used for full path.

Same below.